docker/lnd/start-lnd-btc.sh

```diff
@@ -33,4 +33,9 @@ if [[ -n "$LND_FEE_RATE" ]]; then
     PARAMS="$PARAMS --bitcoin.feerate=$LND_FEE_RATE"
 fi
 
+if [[ -n "$BACKUP_DIR" ]]; then
```

Comment:
Are there cases where we won't specify this?

Reply:
I cant think of any when we (sparkswap) wouldn't but if it is left out, itll just default to the `data` dir.

Reply:
It might pay to be a bit more opinionated here and not support not customizing this. Would avoid some weird issues where this value is not defined by mistake.